src/demo/App.js

```diff
@@ -131,6 +131,7 @@ const tourConfig = [
     content:
       "Ok, let's start with the name of the Tour that is about to begin.",
     actionBefore: async () => {
+      // TODO: This should be moved to another step of the demo to show that the step can wait for animations to complete
```

Comment:
Could you explain a little more?

Just to contextualize a bit, because of this _feature_ I [already didn't close the next version](https://github.com/elrumordelaluz/reactour/issues/102#issuecomment-577063963)

Reply:
The first step of the demo tour is not the place to use `await timeout(5000)` it might give any prospective developers looking to use reactours the impression its not performant. This is mainly because the dialog box does not attach to the target element until after the timeout is completed.

Reply:
absolutely agree, however I want to add an _async_ step example in the future.